text/symbols.talon

```diff
@@ -29,6 +29,10 @@ empty escaped string:
     "\\'\\'"
     key(left)
     key(left)
+empty square:
+  insert("[]")
```

Comment:
What would you use these two commands for?

Reply:
During variable assignment of empty arrays and empty objects

Reply:
The other empty* commands seem to place the cursor between the inserted wrappers. From what you say it sounds like this wouldn't work for you (and the 'empty square' at least would be redundant with 'op subscript'). I'm thinking to keep things consistent we leave these two out of knausj?